internal/array/sort.go

```diff
@@ -22,3 +22,9 @@ func Sort2D(slice [][]string) {
 		})
 	}
 }
+
+func SortStrings(slice []string) []string {
```

Comment:
Does this need to be a function?

Reply:
I would just remove this and call sort.Strings directly where you need it, I don't see the point in wrapping this.